Dockerfile

```diff
@@ -3,7 +3,19 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ############# golang-base
-FROM eu.gcr.io/gardener-project/3rd/golang:1.16.2 AS golang-base
+FROM golang:1.16.2 AS golang-base
+
+############# terraform-config
+FROM golang-base AS terraform-config
+
+# overwrite to build provider-specific image
+ARG PROVIDER=all
+
+# Get entire dir to preserve the symblic links to TF_VERSION files
+COPY . .
+
+RUN cp ./build/$PROVIDER/TF_VERSION /TF_VERSION && \
+    cp ./build/$PROVIDER/terraform-bundle.hcl /terraform-bundle.hcl
```

Comment:
Thanks for the follow-up on this issue.
Another simpler option would be to remove the symlinks and each provider under `build/` to maintain its own TF_VERSION file? WDYT?

Reply:
Well, there are multiple options. I even have prepared another solution that was falling back to the `TF_VERSION` in the root of the repo. However, I like that with symbolic links we are maintaining the version just on one place, therefore I would like to preserve them.

Reply:
I have removed the symlinks and each provider has its own regular TF_VERSION file.

Reply:
Thanks! I assume we can also remove the TF_VERSION file in the repo root (and the reference in `.dockerignore`)?

Reply:
Thanks! Also done.